ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vxlan-interface.md

```diff
@@ -65,13 +65,15 @@ interface Management1
 
 ### VXLAN Interface Summary
 
-#### Source Interface: Loopback0
-
-#### MLAG Source Interface: Loopback1
-
-#### UDP port: 4789
-
-#### EVPN MLAG Shared Router MAC : mlag-system-id
+| Settings | Values |
+| -------- | ------ |
+| Source Interface | Loopback0 |
+| MLAG Source Interface | Loopback1 |
+| UDP port | 4789 |
+| EVPN MLAG Shared Router MAC | mlag-system-id |
+| VXLAN flood-lists learning from data-plane | Enabled |
+| Qos dscp propagation encapsulation | Enabled |
+| Qos map dscp to traffic-class decapsulation | Enabled |
```

Comment:
@ClausHolbechArista should these settings be using proper case or just capitalize the first word ?